testsuite/integration-tests/src/test/java/org/jboss/resteasy/test/providers/iioimage/IIOImageProviderTest.java

```diff
@@ -70,8 +70,8 @@ private static String generateURL(String path) {
     * @tpSince RESTEasy 3.0.16
     */
    @Test
-   @Ignore("[RESTEASY-1724] java.lang.IllegalStateException: Compression mode not MODE_EXPLICIT!")
    public void testPostJPEGIMage() throws Exception {

```

Comment:
Can we change the method name too? There is a little mismatch JPEG vs PNG ;)

Reply:
method name changed